datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/chart/mappers/InputFieldsMapper.java

```diff
@@ -31,13 +35,19 @@ public com.linkedin.datahub.graphql.generated.InputFields apply(
             .map(
                 field -> {
                   InputField fieldResult = new InputField();
+                  Urn parentUrn = entityUrn;
 
-                  if (field.hasSchemaField()) {
-                    fieldResult.setSchemaField(
-                        SchemaFieldMapper.map(context, field.getSchemaField(), entityUrn));
-                  }
                   if (field.hasSchemaFieldUrn()) {
                     fieldResult.setSchemaFieldUrn(field.getSchemaFieldUrn().toString());
+                    try {
+                      parentUrn = Urn.createFromString(field.getSchemaFieldUrn().getEntityKey().get(0));
+                    } catch (URISyntaxException e) {
+                      log.error("Field urn resolution: failed to extract parentUrn successfully from {}. Falling back to {}", field.getSchemaFieldUrn(), entityUrn, e);
```

Comment:
What is the case where this was happening?

Reply:
I guess nowhere, just defensive

Reply:
I see - this is where the bad mapping was happening. Got it

Reply:
Mainly a protection from bad data - in case we have mal formatted schema field urns.